allennlp_server/tests/server_simple_test.py

```diff
@@ -31,7 +28,8 @@ def setUp(self):
         super().setUp()
 
         print(os.getcwd())
-        archive = load_archive("tests/fixtures/bidaf/model.tar.gz")
+        importlib.import_module("allennlp_rc.models")
```

Comment:
This is used by `load_archive`, to find the "bidaf" model.

I think it's neater to load this like this, instead of the vanilla `import`, because the entity/object `BidirectionalAttentionFlow` is not used anywhere here, thus it's like the import it's not used. So I believe it's better to load it "internally", with `importlib`.